WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1180,7 +1180,7 @@ InviteStatusCounterBgColor  = #fff4d5
 InviteStatusCounter         = background-color: @InviteStatusCounterBgColor@; margin: @MailViewGutterSpace@ 0;
 InvCounter                  = padding:0 24px;
 
-ComposeViewContainer        = @PageBg@ @FullWidth@ @Conv2Bg@; box-sizing: border-box; padding: @MailViewGutterSpace@; padding-top: 0;
+ComposeViewContainer        = @PageBg@ @FullWidth@ @Conv2Bg@; box-sizing: border-box; padding: @MailViewGutterSpace@; padding-top: 0; border-top: 1px solid @PanelBorderColor@;
```

Comment:
@PanelBorder@ is for border from all side, and here only border for top is styled. So not needed, I think.

Thanks